### DISCUSSION TITLE
Error: Failed to read source code from ./src/app/contact/error.jsx - "The system cannot find the file specified" (os error 2)

### DISCUSSION THREAD

**Discussion by Redbeardjunior** [vercel/next.js#51317] [App Router]
Question: I'm encountering an error in my ReactJS project after I deleted the file "./src/app/contact/error.jsx". The error message I'm receiving is as follows:

Error: Failed to read source code from C:\Users...\Desktop\reactjs\src\app\contact\error.jsx

Caused by: The system cannot find the file specified. (os error 2) This error indicates that the system is unable to locate the file, resulting in an operating system error 2. It's important to note that I intentionally deleted the error.jsx file from the specified path.

![image](https://github.com/vercel/next.js/assets/16633643/133b9195-eefd-4f8b-a1a2-46668629f66c)

Here are the details of my development environment:

Operating System: windows 11

React version: 13

I have tried the following steps to troubleshoot the issue, but none of them have resolved the error:

File restoration: I attempted to restore the deleted error.jsx file from backup or recycle bin, but was unsuccessful. Clean and rebuild: I cleaned and rebuilt the project to ensure any cached references to the file were cleared, but the error persisted. Dependency checks: I verified that there are no dependencies or modules still referencing the deleted file. Despite these attempts, I am still encountering the same error. It seems that the project is still trying to access the deleted file, causing the system to throw an error. I'm unsure how to resolve this issue and would appreciate any guidance or insights from the community. Thank you!

when im looking for a call to the file, I can't find any..

![image](https://github.com/vercel/next.js/assets/16633643/85ebc272-26bd-42bb-ab8e-185bfa23bb3e)

**Comment by market-south** [comment]
Hi 

Have you tried to delete the `.next` folder and tried to rebuild and/or just search for `"error"` in your IDE?  I had this issue in the past, deleting `.next` and rebuilding the project resolved the issue.

Hope this helps

**Reply by Redbeardjunior** [reply]
Thank you ! is there any reason why this happens or is thiss because dev mode is running and you delete something while running ?
